health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/securityPopUp.js

```
@@ -46,79 +55,95 @@ const SecurityPopUp = ({ onClose, census }) => {
 
   // Function to save answers and trigger API call
   const handleSave = async () => {
-    await mutation.mutate();
+    await mutation.mutate(
+      { body: updatedCensus },
+      {
+        onSuccess: (data) => {
+          onSuccess && onSuccess(data); // Call the onSuccess callback if provided
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)